semconv/v1.4.0/http_test.go

```diff
@@ -885,6 +904,8 @@ func getExpectedCodeForHTTPCode(code int) codes.Code {
 	category := code / 100
 	if category > 0 && category < 4 {
 		return codes.Unset
+	} else if spanKind == trace.SpanKindServer && category == 4 {
+		return codes.Unset
```

Comment:
Just a minor code style nit.  The `else` doesn't really add anything here. You could probably just change this to
```
if category > 0 && category < 4 {
    return codes.Unset
}
if spanKind == trace.SpanKindServer && category == 4 {
    return codes.Unset
}
return codes.Error
```

Reply:
done. thanks for your advice.